layouts/_default/_markup/render-heading.html

```diff
@@ -0,0 +1,3 @@
+{{/*  enable heading self links */}}
+{{ template "_default/_markup/td-render-heading.html" . }}
```

Comment:
would be good to include link to the docs here as a magical file reference to docsy things: 

https://www.docsy.dev/docs/adding-content/navigation/#heading-self-links